build.yml

```diff
@@ -111,7 +114,8 @@ jobs:
 
       - bash: |
           set -ex
-          conda build $VERBOSE_OPTION \
+          source activate base
+          boa build $VERBOSE_OPTION \
```

Comment:
```suggestion
          conda mambabuild $VERBOSE_OPTION \
```

Reply:
described in the Readme here https://github.com/mamba-org/boa#boa-the-fast-build-tool-for-conda-packages